scripts/lint.py

```diff
@@ -170,6 +173,14 @@ def lint_line(
         if re.search(r"\b3d\b", line):
             return "we prefer '3D' over '3d'"
 
+    if (
+        "recording=rec" in line
+        and "rr." not in line
+        and "recording=rec.to_native()" not in line
+        and "recording=recording.to_native()" not in line
+    ):
+        return "you must cast the RecordingStream first: `recording=recording.to_native()"
```

Comment:
:see_no_evil: this will surely bite us at some point... but this will be fixed when we get rid of the wrapper types all together